deploy/ocpbugs-15043/10-ocpbugs-15043.CronJob.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: ocpbugs-15043
+  namespace: openshift-config
+spec:
+  failedJobsHistoryLimit: 2
+  successfulJobsHistoryLimit: 2
+  concurrencyPolicy: Forbid
+  schedule: "*/5 * * * *"
```

Comment:
what's motivating the 5-minute schedule?

Reply:
Just as a starting point for a discussion :) I wasn't sure about the interval so i just picket 5 minutes to begin with. Ideally, this cronjob is faster at bouncing faulty machines than our alerting is at detecting the failed pods.